src/ui/main_menu.tscn.tmp

```diff
@@ -0,0 +1,57 @@
+[gd_scene load_steps=3 format=2]
```

Comment:
this file seems to be a Godot-generated temporary file. We should add `**.tmp` to our `.gitignore` file.